src/components/Trade/hooks/useSwapper/utils.ts

```diff
@@ -96,9 +96,9 @@ export const getFirstReceiveAddress: GetFirstReceiveAddress = async ({
   const { chainId } = buyAsset
   const accountId = toAccountId({ chainId, account })
 
-  const { accountType, utxoParams } = accountIdToUtxoParams(accountId, 0)
-
-  return await chainAdapter.getAddress({ wallet, accountType, ...utxoParams })
+  const { accountType } = accountIdToUtxoParams(accountId, 0)
+  const bip44Params = chainAdapter.getBIP44Params({ accountNumber: 0 })
```

Comment:
plz add big TODO here to remove this

Reply:
Captured in https://github.com/shapeshift/web/issues/2730